src/main/java/ch/njol/skript/effects/EffLog.java

```diff
@@ -77,12 +82,15 @@ public void close() {
 	private Expression<String> messages;
 	@Nullable
 	private Expression<String> files;
-	
+
+	private int loglevel;
```

Comment:
instead of storing the log level as an int, you can store the `Level` itself to pass to the logger